start

```diff
@@ -271,8 +271,12 @@ stack_snapshot(
     name = "stackage",
     extra_deps = {"zlib": ["@zlib.dev//:zlib"]},
     packages = ["zlib"],
-    snapshot = "lts-14.27", # Last snapshot published for ghc-8.6.5
-    # the default version picked up by rules_haskell
+    # Last snapshot published for ghc-8.6.5 the default version picked up by
+    # rules_haskell
+    snapshot = "lts-14.27",
+    # Pin Stackage dependencies with \`bazel run @stackage-unpinned//:pin\`.
```

Comment:
Similar to my comment in `examples/WORKSPACE`.

Reply:
done